tests/fixtures/sample_lib/modules/other_task/other_implementation.py

```diff
@@ -54,7 +54,7 @@ def train(
         batch_size: int = 64,
     ) -> "OtherModule":
         """Sample training method that produces a trained model"""
-        assert type(sample_input) == SampleInputType or str
+        assert type(sample_input) == (SampleInputType or str)
```

Comment:
This was definitely wrong before, but I'm not sure this is correct either?
```
>>> str == (int or str)
False
```

Reply:
lol yeah 🤦

Reply:
I'm going to go with
```
isinstance(sample_input, SampleInputType) or isinstance(sample_input, str)
```

Reply:
We do have the type checker if you wanted to use that, but I don't really mind either way